File: 26-RemoveDuplicatesfromSortedArray/26-RemoveDuplicatesfromSortedArray.py
# Last updated: 4/21/2026, 10:10:55 AM
1class Solution:
2    def removeElement(self, nums: List[int], val: int) -> int:
3        k = 0
4        for i, n in enumerate(nums):
5            if n != val:
6                nums[k] = n
7                k += 1
8        return k

File: 26-RemoveDuplicatesfromSortedArray/26-RemoveDuplicatesfromSortedArray.cpp
// Last updated: 9/11/2026, 4:17:47 PM
1class Solution {
2public:
3    int removeDuplicates(vector<int>& nums) {
4        int k = 1;
5        for (int i = 1; i < nums.size(); ++i) {
6            if (nums.at(i) != nums.at(i - 1)) nums.at(k++) = nums.at(i);
7        }
8        return k;
9    }
10};